### PR TITLE
Update admin playbooks

### PR DESCRIPTION
**Pull request by hexylena**

### CODE REVIEW

**Review by shiltemann — APPROVED** [review]
thanks @hexylena

File: topics/admin/tutorials/ansible-galaxy/group_vars/galaxyservers.yml [hunk 1]
```diff
@@ -18,20 +18,23 @@ galaxy_manage_paths: true
...
 galaxy_root: /srv/galaxy
 galaxy_user: {name: galaxy, shell: /bin/bash}
-galaxy_commit_id: release_20.05
+galaxy_commit_id: release_20.09
 galaxy_config_style: yaml
 galaxy_force_checkout: true
+
 miniconda_prefix: "{{ galaxy_tool_dependency_dir }}/_conda"
 miniconda_version: "4.6.14"
+miniconda_manage_dependencies: false
```
Comment (nsoranzo) [inline topics/admin/tutorials/ansible-galaxy/group_vars/galaxyservers.yml:27]: @hexylena That's not mentioned in the tutorial AFAICS, which one is right?

Comment (hexylena) [reply topics/admin/tutorials/ansible-galaxy/group_vars/galaxyservers.yml:27]: that should be there yea. it should be mentioned. this commit got forgotten and added, its been done in several beanches. will fix in jamuary

Comment (hexylena) [reply topics/admin/tutorials/ansible-galaxy/group_vars/galaxyservers.yml:27]: thnaks for the PR to fix this!

**Pull request merged**